CONTRIBUTING.md

```diff
@@ -88,6 +88,10 @@ When working on a new feature and code changes, it's important that things work
 
 We have snapshot tests to ensure that components output the expected markup. If you change a `render` function, it will likely produce some snapshot failures when running `yarn test`. If the snapshot changes are what you expect, you can run `yarn test -u` to update the snapshots. Don't forget to include them in your commit as well.
 
+### Updating locales
```

Comment:
This shouldn't be a 3rd level title (it's not part of "Running the tests"), it should be a 2nd level title.

With that said, I would keep it out of this pull request, and file a separate issue.

The text would need to cover a bit more (where localization happens, how IDs are formed, do not touch other locales even to remove strings, etc.)

Reply:
Yeah +1 for me too